sources/packages/backend/workflow/test/2023-2024/fulltime-assessment/eligibility/fulltime-assessment-2023-2024-eligibility-BGPD.e2e-spec.ts

```diff
@@ -21,7 +21,7 @@ describe(`E2E Test Workflow fulltime-assessment-${PROGRAM_YEAR}-eligibility-BGPD
         // Arrange
         const assessmentConsolidatedData =
           createFakeConsolidatedFulltimeData(PROGRAM_YEAR);
-        assessmentConsolidatedData.studentPDStatus = true;
+        assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
```

Comment:
We have a `YesNoOptions` enum available for the workflows E2E here
https://github.com/bcgov/SIMS/blob/main/sources/packages/backend/libs/test-utils/src/models/common.model.ts#L20-L23

Reply:
ya, but here there is no direct `no`, 
![image](https://github.com/bcgov/SIMS/assets/77353155/cbff9320-96f6-43f5-90f4-b72681457115)
that's why I used string